build.gradle

```diff
@@ -29,5 +30,31 @@ test.finalizedBy("checkstyleMain")
 
 task runHelloWorldJoseG (dependsOn: 'classes', type: JavaExec) {
     main = 'jalau.at18.katas.helloworld.joseg.Main'
+    standardInput=System.in
```

Comment:
This line might not be needed, you can remove it

Reply:
Line has been removed